scripts/griddle.jsx

```diff
@@ -649,8 +650,13 @@ var Griddle = React.createClass({
         }
     },
     _toggleSelectAll: function () {
-        var visibleRows = this.getDataForRender(this.getCurrentResults(), this.columnSettings.getColumns(), true),
-            newIsSelectAllChecked = !this.state.isSelectAllChecked,
+        var visibleRows = [];
+        if (this.props.enableInfiniteScroll) {
+            visibleRows = this.getCurrentResults();
```

Comment:
It looks like `getDataForRender` is doing a lot of stuff, can we really just drop it for infinite scroll?